### PR TITLE
Use database

### PR DESCRIPTION
**Pull request by sgreensmith**

### CODE REVIEW

**Comment by SKeerthana** [comment]
There is WIP commit message. Can we change the message before merging?

File: dev-deployment.sh [hunk 1]
```diff
@@ -31,9 +32,22 @@ cfLogin() {
...
+
+cfBindWithDatabase() {
+    cf bind-service $APP_NAME verify-local-matching-service-example-db
+    cf restart $APP_NAME
+}
+
 ./docker-pre-commit.sh
 
-cfLogin
+#cfLogin
```
Comment (RachaelBooth) [inline dev-deployment.sh]: This should presumably not be commented out?

File: database-schema.sql [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    address INTEGER,
+    date_of_birth DATE,
+    FOREIGN KEY(address) REFERENCES address(address_id)
+);
+
+CREATE TABLE verifiedPid (
+    pid VARCHAR PRIMARY KEY,
+    person INTEGER,
+    FOREIGN KEY(person) REFERENCES person(person_id)
+);
```
Comment (RachaelBooth) [inline database-schema.sql]: We should note somewhere, although I'm not sure where, that this needs to be kept up to date with the example local matching service (assuming we have no way to directly get the current database structure from there).

**Comment by RachaelBooth** [comment]
Could we update the readme to mention the database?

File: dev-deployment.sh (hunk 1 above)
Comment (sgreensmith) [reply dev-deployment.sh]: oh yes, good catch.

**Pull request merged**